src/lib.ts

```diff
@@ -153,7 +153,7 @@ export function logTestSetup(whichTest: CommunityService): void {
 export const mainStatMaximizerString = myPrimestat() === $stat`Muscle` ? "mus" : myPrimestat() === $stat`Mysticality` ? "myst" : "mox";
 
 //Define how to determine mainstat and define certain effects, incrediants, and reagant needs based on mainstat
-export const mainStatStr = myPrimestat().toString();
+export const mainStatStr = myPrimestat();
```

Comment:
Variable names should be descriptive of what they are. I keep harping on this, and albeit being a minor point (even if the code compiles with bad naming schemes), it's a good habit to keep practicing it.

`const mainStat` is descriptive of a (const) variable that is of type `$stat` (e.g. `$stat'Muscle'`).
`const mainStatStr` is descriptive of a (const) variable that is the string of the main stat (e.g. `"Muscle"`).
`const mainStatMaximizerString` is descriptive of a (const) variable that is the string of the main stat used in the maximizer (e.g. `"Mus"`)
(we can ignore for now that we are using `Str` in one variable name and `String` in the other - ideally they should also be standardized)

Checking `mainStatMaximizerString === $stat'Muscle'` does not make sense to the reader.
Neither does `mainStatStr === $stat'Muscle'`.

Work with the following exported consts below and do not change them any further.
```ts
export const mainStat = myPrimestat();
export const mainStatStr = mainStat.toString();
```
You should **not** be calling `.toString()` anywhere else once you have this.

If VSCode throws out red squiggly lines, you are using the wrong exported const. Choose the correct one instead of redefining the existing one to fit your purposes - again the consts are named what they are because they are descriptive of what they contain.

Reply:
Understood. Updated.